src/applications/mhv-landing-page/sass/mhv-landing-page.scss

```diff
@@ -14,6 +14,17 @@ $anchor-font-size: 1.25rem; // == 20px
   }
 }
 
+$mhv-desktop-screen: 1008px;
```

Comment:
Suggestion: Maybe move this to the top of the file with the other variable defined there.